unit-tests/test-fw-update.py

```diff
@@ -3,7 +3,8 @@
 
 # we want this test to run first so that all tests run with updated FW versions, so we give it priority 0
 #test:priority 0
-#test:device each(D400*) !D457
+#test:donotrun:gha
```

Comment:
Why is this needed?
Did it tried to run on GHA until today?

Reply:
Eran have asked me to add this. The test tried to run as part of all the tests, there was no Acroname so it exited, not relevant for GHA